main_test.go

```diff
@@ -31,9 +33,9 @@ func setupIntegrationTests() {
 	os.Setenv(coreutils.ReportUsage, "false")
 	// Disable progress bar and confirmation messages.
 	os.Setenv(coreutils.CI, "true")
-
 	flag.Parse()
 	log.SetDefaultLogger()
+	validateAliasesDuplication()
```

Comment:
validateAliasesDuplication --> validateCmdAliasesUniqueness

(the existing name make it sound like we want to find duplications).